app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class StaticPagesController < ApplicationController
+	before_action :redirect_to_activity_if_signed_in
```

Comment:
Does this mean that the URL changes when a loggedin user visits root path?
If so, that is a bad experience.